### ISSUE TITLE
error: Module has no attribute "tzlocal"

### ISSUE THREAD

**Issue by walles** [python/mypy#4174]
Hi!

Do `pip install 'python-dateutil == 2.6.1'`

Then take this program:
```py
import dateutil.tz
print(dateutil.tz.tzlocal())
print(dateutil.tz.tzlocal)
```

And run mypy 0.540:
```
(env) /p/tmp $ mypy repro.py
repro.py:2: error: Module has no attribute "tzlocal"
repro.py:3: error: Module has no attribute "tzlocal"
```

But the program runs just fine:
```
(env) /p/tmp $ python3 repro.py
tzlocal()
<class 'dateutil.tz.tz.tzlocal'>
```

Note that if you run `mypy` in Python 2.7 mode it doesn't complain:
```
(env) /p/tmp $ mypy --python-version=2.7 repro.py
(env) /p/tmp $
```

My expectation would be for mypy not to complain about this program, even when run in Python 3 mode.

  Regards /Johan

**Comment by ethanhs** [comment]
This was an error in the way typeshed stubs were exporting the names for dateutil that I fixed in https://github.com/python/typeshed/commit/b7dc041f446e461db918ffa5127b624f883081c2. It now available in mypy master with #4171. Additionally, it will be fixed in the next release.

**Comment by walles** [comment]
Thanks for the quick turnaround @ethanhs!
